pkg/adapter/sigproxy_linux.go

```diff
@@ -20,7 +20,9 @@ func ProxySignals(ctr *libpod.Container) {
 		for s := range sigBuffer {
 			// Ignore SIGCHLD and SIGPIPE - these are mostly likely
 			// intended for the podman command itself.
-			if s == syscall.SIGCHLD || s == syscall.SIGPIPE {
+			// SIGURG was added because of golang 1.14 and its preemptive changes
+			// causing more signals to "show up".
```

Comment:
Can you link https://github.com/containers/libpod/issues/5454 in here so in the future we know what's going on?

Reply:
done